testsuite/systemtest/bintools/Compiler/tcetest_double_support.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+### TCE TESTCASE
+### title: Test double-precision operations and operands
+
+#old:mach=data/multi_addrspace.adf
+#old:src=data/multi_addrspace.c
+mach=data/doubles.adf
+src=data/doubles.cc
+program=$(mktemp tmpXXXXXX)
+
+#set -x
+tcecc -a $mach -O3 $src -o $program && \
+ttasim -a $mach -p $program --no-debugmode
+
```

Comment:
Absolutely.  This is a starter shell at the moment (obviously since it wasn't even executable before).  I will add lots more here.  For example, by emitting the LLVM code I can check those "//rwh: check" node generators in TDGen.cc.